src/scripts/objects/SvgTip.js

```diff
@@ -73,7 +73,7 @@ export default class SvgTip {
 
 		this.top = this.y - this.container.offsetHeight;
 		this.left = this.x - width/2;
-		let max_left = this.parent.offsetWidth - width;
+		let max_left = window.innerWidth - (width/2);
```

Comment:
So this would affect all tooltips. How do they behave after this?

Reply:
Both this and fixed positioning affect other charts. I tested on `docs/index.html` and all chart types looked ok.